whitepapers/game-theory.md

```diff
@@ -12,6 +12,13 @@ from the perspective of analysing human interaction.
 There are many fields that have something to say about human interaction but the
 field of study we will focus on here will be game theory - the study of strategic interdependence.
 
+It is our belief that Olympus is solving the problem of how to create a new currency without government prescribed use of force. At its core this is an example of a [prisoner's dilemma](https://en.wikipedia.org/wiki/Prisoner%27s_dilemma). A prisoner's dilemma is a situation where an individual's self interest is in conflict with a common goal, leading to the players within the game not cooperating despite it being in their best interests to cooperate. 
+
+In the case of currency, it is in each individual's best interest to use the most liquid, most widely used and most stable currencies. In crypto the assets that best meet these requirements have been dollar denominated stablecoins. The ‘common goal’ in this context, is to remove reliance on centralized fiat currency. This is something which stablecoins categorically do not achieve and yet still they are some of the most used assets in crypto. No individual acting alone can disentangle crypto from fiat. It takes mass coordination and alignment of incentives and this is what Olympus facilitates.   
+
+We will begin by outlining the essentials of game theory, and analysing the prisoner's dilemma from a purely abstract perspective. We will then delve into the specific components of Olympus. We will take no shortcuts. Olympus is a complex protocol that is the first of its kind and is deserving of deep and thorough analysis. 
```

Comment:
```suggestion
We will begin by outlining the essentials of game theory, and analysing the prisoner's dilemma from a purely abstract perspective. We will then delve into the specific components of Olympus. We will take no shortcuts. Olympus is a complex protocol that is the first of its kind and is deserving a deep and thorough analysis. 
```

Reply:
I think my original phrasing makes more sense. 
Could re-phrase slightly to;
'Olympus is a complex protocol that is the first of its kind and is deserving of a deep and thorough analysis.'